Contents/Code/__init__.py

```diff
@@ -278,33 +264,33 @@ def update(self, metadata, media, lang):
 			for x in range(len(metadata.art)):
 				for item in metadata.art:
 					t = item
-					break;
+					break
 				del metadata.art[t]  
     
-		num_arts = Prefs['numthumb'];
-		i = 0;
+		num_arts = Prefs['numthumb']
+		i = 0
 		sort = 1
-		for arts in info_filmweb_art_result.xpath(".//ul[contains(@class, 'photosList')]/li"):
+		for arts in info_filmweb_art_result.xpath(".//ul[@class='gallery__photos-list grid']/li/a[1]/@data-photo"):
```

Comment:
Brak przypisania do zmiennej info_filmweb_art_result (ale to raczej mój stary błąd) :)

info_filmweb_art_result = HTML.ElementFromURL(FILMWEB + movie_id + '/photos', 0, {'Cookie':HTTP.CookiesForURL(FILMWEB)}, cacheTime = 0)

Reply:
Jest w obu wersjach ;) w mojej w wierszu 260.